calicoctl/calico_ctl/node.py

```diff
@@ -262,8 +262,10 @@ def node_start(node_image, runtime, log_dir, ip, ip6, as_num, detach,
     if not os.path.exists(log_dir):
         os.makedirs(log_dir)
 
-    # The format of the authority string has already been validated.
+    # The format of the authority and endpoints strings have already been
+    # validated.
     etcd_authority = os.getenv(ETCD_AUTHORITY_ENV, ETCD_AUTHORITY_DEFAULT)
+    etcd_endpoints = os.getenv(ETCD_ENDPOINTS_ENV, None)
```

Comment:
We pass , None in many other places.  I actually prefer leaving it; I think it's faster to read with the Nones than forcing people to pull the default from memory (/ the internet).